package.json

```diff
@@ -36,6 +36,7 @@
     "selenium-webdriver": "2.45.1"
   },
   "devDependencies": {
+    "angular-history": "git+https://github.com/decipherinc/angular-history.git#v0.8.0",
```

Comment:
not 100% sure, but imo this should be part of dependencies, because you need it installed also when using this repo as library.